src/core/constructs/workspace.py

```diff
@@ -100,24 +100,25 @@ class Workspace_State(str, Enum):
     EXECUTING_BACKEND = "EXECUTING_BACKEND"
 
 
+# ANIBAL: I would rename this into a more obvious
+# require_phase or something like that
```

Comment:
I think `wrap_requred_phase` would work with me. I want to leave the `wrap` part so that it is clear that it is a higher order function